workers/unity/Assets/Playground/Scripts/Cubes/CollisionProcessSystem.cs

```diff
@@ -0,0 +1,69 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Playground
+{
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    internal class CollisionProcessSystem : ComponentSystem
+    {
+        private struct LaunchableData
+        {
+            public readonly int Length;
+            public ComponentDataArray<CollisionComponent> Collision;
+            public EntityArray Entities;
+            [ReadOnly] public ComponentDataArray<CommandRequestSender<SpatialOSLauncher>> Sender;
```

Comment:
I'm not a little confused about what this entity is. I though at first it was a spatial one, but now it looks like it's not. In which case this sender won't be matched against anything.